src/main/resources/assets/thaumicaugmentation/lang/en_us.lang

```diff
@@ -595,9 +595,16 @@ thaumicaugmentation.research.vis_battery_augment.1=Although it does not feel lik
 thaumicaugmentation.research.thaumium_robes.1=Thaumium has proven itself to be an excellent metal to work with, meshing well with enchanted fabric. I now have a set of infusion recipes for the Robes of the Adept, including a hood with built-in Goggles of Revealing, a chest-piece and a pair of leggings. I have left out the boots as I feel I may be able to enhance them in a very different way, to allow them to do more than just protect me.<BR>Regardless, these new robes offer far better protection than before, with the additional thaumium plates being able to absorb both physical and magical attacks. Furthermore, the robes also have an improved vis discount on them, allowing me to cast spells and craft magical items just a little more efficiently. They also accept enchantments very well, again due to the thaumium used to make them.<BR>Finally, I have improved the robes’ visual design and I can even recolour them if I so wish. To do this, I simply have to craft the robes with dye(s) in a crafting window, and to wash the dye out I just have to use the robes on a filled, mundane cauldron.
+
+thaumicaugmentation.research.glass_tube.title=Glass Essentia Tube
+thaumicaugmentation.research.glass_tube.0=Essentia piping systems are a major step forward in thaumaturgic alchemy, but this major step forward comes at the price of additional complexity. Managing tube systems can easily become confusing, especially since I cannot easily tell at a glance where and when essentia is flowing, what with all tube types being fully opaque.<BR>This was not done by choice, as previously was not able to find a way to make a pipe primarily made of glass survive the constant barrage of magical essence. However, I may be able to revisit my previous studies and find a way to make glass resistant to even the most volatile of essentia. Perhaps I should look into warded jars as a starting point.
+thaumicaugmentation.research.glass_tube.1=After some further investigation into essentia and piping systems that use it, I was able to rework my designs for ordinary essentia tubes into a new one. With some additional warding magic and iron to seal any remaining gaps, I now have plans ready for a special type of essentia tube that is fully see-through but resistant to essentia leaking or melting through it.<BR>Glass Essentia Tubes work in exactly the same way as ordinary essentia tubes, but their transparent nature means I can see what essentia is flowing through them without needing to use an essentia resonator. This may be useful if I am struggling to understand what my piping systems are doing.<BR>I was not able to make glass versions of my more complex tube types, however. The various mechanisms, valves and such used in them need to be contained by more sturdy metal. However, all of my other tube types do still connect to glass tubes, and as before I can connect and disconnect glass tube segments using my caster’s gauntlet.
```

Comment:
Did we ever decide if the crafting recipe would use any of: fortified glass, warding sigil, etc?

Reply:
The actual wording / writing here is fine, I am just unsure which one to go with, if any.

Reply:
Warding sigils are mainly used for personal blocks rather than stronger blocks, and as you previously mentioned it may be too much hassle for the player who wants to focus on alchemy to have to deal with fortified glass. Therefore, I'd personally vouch for just using regular glass for the sake of simplicity, but you can go with whatever makes the most sense to you.